mathics/builtin/numbers/constants.py

```diff
@@ -348,7 +348,7 @@ class EulerGamma(_MPMathConstant, _NumpyConstant, _SympyConstant):
      = 0.5772156649015328606065120900824024310422
     """
 
-    summary_text = "Euler`s constant"
+    summary_text = "Euler's constant γ ≃ 0.577"
```

Comment:
Personally  I had been shying away from assuming that unicode. But at this point it is not worth any effort thinking about it. Whatever.